mlrun/__main__.py

```diff
@@ -98,9 +98,10 @@ def main():
     help="how to select the best result from a list, e.g. max.accuracy",
 )
 @click.option(
-    "--tuning-strategy",
-    default="",
-    help="hyperparam tuning strategy list | grid | random",
+    "--strategy", default="", help="hyperparam tuning strategy list | grid | random",
```

Comment:
Why changing the name ?

Reply:
it appears in both names (strategy and tuning-strategy) in different places, so prefer to align all on one name, its hardly used today so not an issue

Reply:
done